docs/deployment/configuration.md

```diff
@@ -178,46 +178,49 @@ Here are the configuration keys, for both containers (environment variables) and
+| task_scheduler:lock_key             | TASK_SCHEDULER__LOCK_KEY                         | task_manager_lock          | Lock key for the scheduler in Redis                 |
+| task_scheduler:interval             | TASK_SCHEDULER__INTERVAL                         | 10000                      | Interval to check new task to do (in ms)            |
+| -                                   | -                                                | -                          | -                                                   |
+| sync_manager:enabled                | SYNC_MANAGER__ENABLED                            | `true`                     | Enable/disable the sync manager                     |
+| sync_manager:lock_key               | SYNC_MANAGER__LOCK_KEY                           | sync_manager_lock          | Lock key for the manager in Redis                   |
+| sync_manager:interval               | SYNC_MANAGER__INTERVAL                           | 10000                      | Interval to check new sync feeds to consume (in ms) |
+| -                                   | -                                                | -                          | -                                                   |
+| expiration_scheduler:enabled        | EXPIRATION_SCHEDULER__ENABLED                    | `true`                     | Enable/disable the scheduler                        |
+| expiration_scheduler:lock_key       | EXPIRATION_SCHEDULER__LOCK_KEY                   | expired_manager_lock       | Lock key for the scheduler in Redis                 |
+| expiration_scheduler:interval       | EXPIRATION_SCHEDULER__INTERVAL                   | 300000                     | Interval to check expired indicators (in ms)        |
+| -                                   | -                                                | -                          | -                                                   |
+| retention_manager:enabled           | RETENTION_MANAGER__ENABLED                       | `true`                     | Enable/disable the retention manager                |
+| retention_manager:lock_key          | RETENTION_MANAGER__LOCK_KEY                      | retention_manager_lock     | Lock key for the manager in Redis                   |
+| retention_manager:interval          | RETENTION_MANAGER__INTERVAL                      | 60000                      | Interval to check items to be deleted (in ms)       |
+| -                                   | -                                                | -                          | -                                                   |
+| notification_manager:enabled        | NOTIFICATION_MANAGER__ENABLED                    | `true`                     | Enable/disable the notification manager             |
+| notification_manager:lock_key       | NOTIFICATION_MANAGER__LOCK_KEY                   | notification_manager_lock  | Lock key for the manager in Redis                   |
```

Comment:
this is not up to date, we should have two lock keys "lock_live_key" and "lock_digest_key" : 

```
"notification_manager": {
    "enabled": true,
    "lock_live_key": "notification_live_manager_lock",
    "lock_digest_key": "notification_digest_manager_lock",
    "interval": 10000
  },
``